ContentPatcher/docs/i18n/zh/author-guide/action-editdata.md

```diff
@@ -0,0 +1,508 @@
+  * [目标字段](#target-fields)
+* [用法](#usage)
+  * [概述](#overview)
+  * [编辑字典](#edit-a-dictionary)
+  * [编辑列表](#edit-a-list)
+  * [移动列表条目](#moving-list-entries)
+  * [编辑模型](#edit-a-model)
+  * [组合操作](#combining-operations)
+* [目标字段](#target-field)
+  * [格式](#format)
+  * [示例](#examples)
+* [参见](#see-also)
+
+## 基本概念<a name="basic-concepts"></a>
+游戏中有许多类型的数据，Content Patcher将其转化成一些常用概念。
+只有理解了本页所述的概念，你才能理解其余部分，所以请不要跳过该部分！
+
+
+### 数据素材<a name="data-assets"></a>
+ _数据素材（data asset）_ 包括从游戏中加载的: 时间地点，角色对话等数据。比如 `Data/Objects` 包括游戏内所有物品的数据。每种数据的格式都在[维基](https://zh.stardewvalleywiki.com/模组:目录)上有阐述。
+
+以下是三种主要的数据素材：
+
+<table>
+<tr>
+<th>类型</th>
+<th>用法</th>
+</tr>
+<tr>
+<td>字典</td>
+<td>
+
+ _字典（dictionary）_ 是包含许多键值对的列表。其中的键必须有唯一ID。同一个字典中的键值对必须写在 `{` 和 `}` 之中。
+
+比如， `Data/Boots` 就是一个包含了许多键值对的字典：
+
+```js
+{
+    //格式是 "键": "值" （所有引号必须用英文的半角符号）
+    "504": "Sneakers/A little flimsy... but fashionable!/50/1/0/0/Sneakers",
+    "505": "Rubber Boots/Protection from the elements./50/0/1/1/Rubber Boots",
+    "506": "Leather Boots/The leather is very supple./50/1/1/2/Leather Boots"
+}
+```
+
+</td>
+</tr>
+<tr>
+<td>列表</td>
+<td>
+
+ _列表（list）_ 是一组没有明确键的非唯一值。列表中的键值对必须写在 `[` 和 `]`之中。
+
+比如， `Data/Concessions` 就是一个典型的列表：
+
+```js
+[
+    {
+       "ID": 0,
+       "Name": "Cotton Candy",
+       "DisplayName": "Cotton Candy",
+       "Description": "A large pink cloud of spun sugar.",
+       "Price": 50,
+       "ItemTags": [ "Sweet", "Candy" ]
+    },
+    // 篇幅有限，其他数据省略
+]
+```
+
+虽然列表没有键，但Content Patcher通常会指定一个字段作为唯一标识符（详见[_编辑列表_](#编辑列表)）。
+
+</td>
+</tr>
+<tr>
+<td>模型</td>
+<td>
+
+ _模型（model）_ 是一种预定义的数据结构。对于内容包来说，它和字典相同，只是你不能添加新字段（只能编辑已有字段）。
+
+</td>
+</tr>
+</table>
+
+### 条目和字段<a name="entries"></a>
+ _条目（entry）_ 是目标数据中的顶层数据块（即字典中的键值对或列表中的值）
+
+比如， `Data/Objects` 中， `"MossSoup": { ...}` 和 `"PetLicense": { ... }`是两个独立的条目：
+```js
+{
+    "MossSoup": {
+        "Name": "Moss Soup",
+        "Type": "Cooking",
+        "Category": -7,
+        "Price": 80,
+        "ContextTags": [ "color_green" ]
+        ...
+    },
+    "PetLicense": {
+        "Name": "Pet License",
+        "Type": "Basic",
+        "Category": 0,
+        "Price": 0,
+        ...
+    },
+    ...
+}
+```
+
+ _字段（field）_ 是条目中的一个子块。在前面的例子中：
+- `"MossSoup": { ... }` 是一个条目；
+- `"Name": "Moss Soup"` 是 `"MossSoup": { ... }`这个条目中的一个字段。
+
+### 使用目标字段改变目标条目<a name="target-fields"></a>
+在上一部分中，我们说条目是“_目标数据中的一个顶层数据块_”。
+目标字段可以让你改变目标数据的含义。
+
+比如，假设我们将目标字段设置为上面的`ContextTags`字段。那么你会看到的数据是这样的：
+```json
+[ "color_green" ]
+```
+/
+这意味着`ContextTags`中的每一个值都是一个条目，所以你可以添加/替换/删除上下文标记，而无需编辑对象数据的其他部分。
+
+([_目标字段_](#目标字段2)会详细介绍这一点)
+
+
+## 用法<a name="usage"></a>
+### 概述<a name="overview"></a>
+`EditData`由`Changes`下的模型组成，其中包含（见下面的示例）：
+
+<dl>
+<dt>必填字段</dt>
+<dd>
+
+你必须同时指定这两个字段：
+
+字段     | 用途
+--------- | -------
+`Action`  | 操作类型。此操作类型设置为`EditData`。
+`Target`  | 需编辑的[游戏素材名](../author-guide.md#what-is-an-asset)（或多个由逗号分隔的素材名），比如`Characters/Dialogue/Abigail`。该字段支持[令牌](../author-guide.md#tokens)，不区分大小写。
```

Comment:
> **en:** This field supports tokens, and capitalisation doesn't matter.
> **zh:** 该字段支持令牌，不区分大小写。

The field is case-insensitive, but the translation seems to say it's the tokens which are? (This also applies to several other fields throughout these docs.)